src/main/resources/com/google/api/codegen/nodejs/discovery_fragment.snip

```diff
@@ -12,7 +12,8 @@
     // ---------------
     // 1. If not already done, enable the {@context.getService.getTitle} and check the quota for your project at
     //    https://console.developers.google.com/apis/api/{@context.getApi.getName}_component/quotas
-    // 2. This sample uses Application Default Credentials for Auth. If not already done, install the gcloud CLI from
+    // 2. This sample uses Application Default Credentials for Auth.
```

Comment:
So it seems like in all languages, item #1 (where it fill the getTitle) can run long for some API. So maybe some smart wrapping there.